packages/safety-suite-api/src/api/models/types.ts

```diff
@@ -886,3 +886,30 @@ export interface Aggregation<GroupValue = unknown> {
   groupIds: number[];
   groupValue?: GroupValue;
 }
+
+export type ExportColumn = {
+  field: string | string[];
+  label: string;
+};
```

Comment:
Remove duplicate definition of `ExportColumn` in `src/legacyApi/models/queries` and replace with an import.